app/services/katello/candlepin/package_profile_uploader.rb

```diff
@@ -0,0 +1,31 @@
+module Katello
+  module Candlepin
```

Comment:
Would changing the Candlepin namespace to Host be more accurate? The package profile doesn't get stored in Candlepin

Reply:
makes sense! Done